presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -746,7 +748,11 @@ private TableScanNode tableScan(String id, String... symbols)
         TpchTableHandle tableHandle = new TpchTableHandle("orders", 1.0);
         return new TableScanNode(
                 new PlanNodeId(id),
-                new TableHandle(new ConnectorId("tpch"), new TpchTableHandle("orders", 1.0)),
+                new TableHandle(
+                        new ConnectorId("tpch"),
+                        new TpchTableHandle("orders", 1.0),
+                        TestingTransactionHandle.create(),
+                        Optional.of(TestingHandle.INSTANCE)),
```

Comment:
nit: Why not do `new TpchTableLayoutHandle(tableHandle, TupleDomain.all())` to be consistent with line 758?

I note I am suggesting the same change as in https://github.com/prestosql/presto/pull/363/commits/47ce7e62491691eb91126c68db09f5eca1e080ff#diff-a2fa83141184ad84b31d8ee64ff9ab5bR749